core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -16,18 +16,23 @@
  * @fileoverview Unit tests for the about page.
  */
 
-import { ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
-import { EventEmitter, NO_ERRORS_SCHEMA, Pipe }
-  from '@angular/core';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { EventEmitter, Pipe } from '@angular/core';
+import { HttpClientTestingModule } from
+  '@angular/common/http/testing';
 
 import { AboutPageComponent } from './about-page.component';
-import { AboutPageConstants } from './about-page.constants';
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
 import { WindowRef } from 'services/contextual/window-ref.service';
 import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { TranslateService } from 'services/translate.service';
-import { UtilsService } from 'services/utils.service';
+import { WindowDimensionsService } from
+  'services/contextual/window-dimensions.service';
```

Comment:
```suggestion
import { WindowDimensionsService } from 'services/contextual/window-dimensions.service';
```

Reply:
we will need to add disable max-len rule to apply this.

Reply:
Nope, max-len is bydefault disabled for imports